src/streamlink_cli/main.py

```diff
@@ -148,35 +148,20 @@ def create_output(formatter: Formatter) -> Union[FileOutput, PlayerOutput]:
         )
 
 
-def create_http_server(*_args, **_kwargs):
-    """Creates an HTTP server listening on a given host and port.
-
-    If host is empty, listen on all available interfaces, and if port is 0,
-    listen on a random high port.
+def create_http_server(host: Optional[str] = None, port: int = 0) -> HTTPOutput:
+    """
+    Create an HTTP server listening on a given host and port.
+    If host is None, listen on all available interfaces.
+    If port is 0, listen on a random high port.
```

Comment:
Should the random port range be specified? `on a port between X and Y`?

Reply:
OS specific stuff, and barely relevant, as it's just a comment for an internal function.

random high port = 1000 - 2^16-1